.github/workflows/docker-publish.yml

```
@@ -33,19 +33,19 @@ jobs:
           echo "IMAGE_NAME=${GITHUB_REPOSITORY,,}" >> "$GITHUB_ENV"
 
       - name: Setup Docker Buildx
-        uses: docker/setup-buildx-action@e468171a9de216ec08956ac3ada2f0791b6bd435 # v3.11.1
+        uses: docker/setup-buildx-action@c47758b77c9364f4d969e8eed60969f55c5a1b96 # v3.12.0
```

YAMLlint (1.37.1)

[warning] too few spaces before comment: expected 2

(comments)